dataprofiler/profilers/historical_profiler.py

```diff
@@ -0,0 +1,423 @@
+import datetime
+import copy
+from multiprocessing.sharedctypes import Value
+from queue import Queue
+from typing import List
+import numpy as np
+import dataprofiler as dp
+from .profile_builder import Profiler
+from .utils import(
+    find_diff_of_dates,
+    find_diff_of_dicts_with_diff_keys,
+    find_diff_of_lists_and_sets,
+    find_diff_of_matrices,
+    find_diff_of_numbers,
+    find_diff_of_strings_and_bools,
+)
+
+class HistoricalProfiler():
```

Comment:
haven't reviewed this but be sure to run `pre-commit` as documented [here](https://github.com/capitalone/DataProfiler#pre-commit)